test/setup/.globals.json

```diff
@@ -1,4 +1,5 @@
 {
+  "__$$GLOBAL_REWIRE_NEXT_MODULE_ID__": true,
```

Comment:
This should be removed - my tests were failing when I used the `babel-rewire-plugin` but now that that is gone this isn't needed. I hadn't seen it before either.